### PR TITLE
feat(css/minifier): convert to and from initial

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

port of https://github.com/cssnano/cssnano/tree/master/packages/postcss-reduce-initial

**BREAKING CHANGE:**

Yes, due AST changes

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_minifier/src/compressor/declaration.rs [hunk 1]
```diff
@@ -496,6 +496,26 @@ impl Compressor {
                 }
                 _ => {}
             }
+
+            let is_initial = if let Some(ComponentValue::Ident(box Ident { value, span, .. })) =
+                declaration.value.get(0)
+            {
+                if value.to_ascii_lowercase() == js_word!("initial") {
```
Comment (kdy1) [inline crates/swc_css_minifier/src/compressor/declaration.rs:503]: Let's use `value.eq_ignore_ascii_case(js_word!("initial"))` instead

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thank you!
`eq_ignore_ascii_case` is faster than comparison after `to_ascii_lowercase`

**Review by kdy1 — COMMENTED** [review]
swc-bump:
 -  swc_atoms
 -  swc_css_ast --breaking

**Comment by alexander-akait** [comment]
WIP on perf :+1:

**Comment by alexander-akait** [comment]
@kdy1 Ready for review

**Review by kdy1 — APPROVED** [review]
Thanks!

**Pull request merged**